### ISSUE TITLE
How to separate BABYLON commands into components?

### ISSUE THREAD

**Issue by punkoffice** [brianzinn/react-babylonjs#1]
How can I do something like this?

<scene>
   <camera/>
   <lights/>
   <box/>
</scene>

Basically I want to be able to create components out of Babylon commands

**Comment by brianzinn** [comment]
I'm not sure if you mean declaratively in .jsx, but that's not the intent of this protect.  If you want everything in markup as components, so not controlling scene creation manually, but with components then check out this project: https://github.com/brochington/Akkad/blob/master/README.md

**Comment by brianzinn** [comment]
That project I linked you to is unsupported and the replacement is not BabylonJS specific.  I have started building components for BabylonJS concepts.  For now I only have the basics: Free camera, Arc Rotate camera, Box, Sphere, Hemispheric light, Ground, Scene.  I've added model loading and a basic behaviour for rotation.  I'd like to add concepts like shadows, animations and actions.  Just a thought experiment until now. Not sure why they're in a different repo, but here are some examples:
https://brianzinn.github.io/create-react-app-babylonjs/
